src/main/webapp/WEB-INF/tags/instructor/course/recoveryCoursesTable.tag

```diff
@@ -25,7 +25,10 @@
   </thead>
   <c:forEach items="${recoveryCourses.rows}" var="recoveryCourse" varStatus="i">
     <tr>
-      <td id="recoverycourseid${i.index}">${recoveryCourse.courseId}</td>
+      <td id="recoverycourseid${i.index}"
+          data-soft-deleted-course-id="${recoveryCourse.courseId}">
```

Comment:
The indentation looks rather strange, can you either keep it in one line or break it up to more lines so we don't see the weird hanging `>`.

For example, here are some styles commonly used:
https://github.com/airbnb/javascript/tree/master/react#alignment

Reply:
I'm actually following the format below. It won't be consistent if it's changed to other forms. Do you prefer to leave it or we change everything instead? In that case many `.tag` files need to be changed also.

Reply:
It can fit on a single line, isn't it?

Anyway, even if we don't place it on a single line there's no need to change other files because there are many more things to do and this is one of the least important ones. Like within the scope that this PR touches there are other issues that we won't be looking into because it's frankly hard to say if they will be important without being able to consider other priorities.

Just to sidetrack a little, if we could write code in a more iterative manner addressing the most important things then every time in a PR we should be able to get a few sub-issues that can be tackled and re-prioritized later on. If we consolidate those issues we can have a better macro view and can prioritize things better too.